paddle/fluid/platform/device_context.h

```diff
@@ -525,6 +525,9 @@ class MKLDNNDeviceContextThreadLocals {
     // Recently registered data_format. This is needed to
     // know for converting MKL-DNN Tensor to non MKL-DNN
     paddle::framework::DataLayout cur_paddle_data_layout;
+    // MKL-DNN stream used for execution of primitives (per-thread)
+    mkldnn::engine aengine;
+    mkldnn::stream astream;
```

Comment:
The names of these variables don't stick to current naming convention in this struct. I'd suggest to remove the `a` prefix.

Reply:
ok